fe/fe-common/src/main/java/org/apache/doris/common/FeMetaVersion.java

```diff
@@ -83,8 +83,11 @@ public final class FeMetaVersion {
     // for java-udtf add a bool field to write
     public static final int VERSION_131 = 131;
 
+    // support db comment
+    public static final int VERSION_132 = 132;
```

Comment:
rebase to newest master, current master's VERSION is 132, so u should use 133

Reply:
@morrySnow  Thank you for your suggestion. I have corrected it in the new pull request: https://github.com/apache/doris/pull/34625